tests/test_auto_lambda.py

```diff
@@ -20,26 +20,12 @@
     SpectrumGenerator
 from trident.testing import \
     answer_test_data_dir, \
+    compare_spectra, \
     TempDirTest
 
 COSMO_PLUS_SINGLE = os.path.join(answer_test_data_dir,
                                  "enzo_cosmology_plus/RD0009/RD0009")
 
-def compare_spectra(sg1, sg2, comp_key):
```

Comment:
This just got moved to another file because I use it for two different tests now.